juicy-filedrop.html

```diff
@@ -83,12 +83,22 @@
                     return false;
                 }
                 if (file.size > this.maxfilesize) {
+                    var maxfilesizeKb = this.maxfilesize / 1024,
+                        maxfilesizeMb = maxfilesizeKb / 1024,
+                        maxfilesizeString = '';
+
+                    if (maxfilesizeKb > 1024) {
+                        maxfilesizeString = '' + parseFloat(maxfilesizeMb).toFixed(2) + 'MB';
+                    } else {
+                        maxfilesizeString = '' + parseFloat(maxfilesizeKb).toFixed(2) + 'KB';
```

Comment:
I think that `parseFloat` is redundant here, because both variables are actually numbers already. Do you agree?